src/DroneSafetyApi/Models/CricularHazard.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.Azure.Documents.Spatial;
+
+namespace DroneSafetyApi.Models
+{
+    public class CircularHazard : Hazard
+    {
+        public new Point Location { get; set; }
```

Comment:
Because we are replacing Geometry Location with Point Location, for more, see  https://msdn.microsoft.com/en-us/library/435f1dw2.aspx